misc/testerina/modules/testerina-runtime/src/main/java/org/ballerinalang/test/runtime/entity/CoverageReport.java

```diff
@@ -124,7 +128,8 @@ public void generateReport(JBallerinaBackend jBallerinaBackend, String includesI
                     jBallerinaBackend, module.packageInstance());
         }
         if (!filteredPathList.isEmpty()) {
-            CoverageBuilder coverageBuilder = generateTesterinaCoverageReport(orgName, packageName, filteredPathList);
+            CoverageBuilder coverageBuilder =
+                    generateTesterinaCoverageReport(orgName, packageName, filteredPathList, duplicateModules);
```

Comment:
Let's introduce an argument to pass the list of Documents to exclude

Reply:
Added a document exclusion list to exlude generated files